exporter/type.go

```diff
@@ -1,14 +1,20 @@
 package exporter
 
-const DefaultClusterName = "default"
+const (
+	DefaultClusterName = "default"
+
+	ComponentLabelKey = "app.kubernetes.io/component"
+	ClusterLabelKey   = "app.kubernetes.io/cluster"
+
+	// FQNamespace represents the prometheus FQName
+	FQNamespace  = "nebula"
+	NonNamespace = "none_namespace"
```

Comment:
none seems more concise

Reply:
I consider concise name may  easy to used by user, so I use a long name `none_namespace`